man/systemctl.xml

```diff
@@ -1647,6 +1648,13 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         dependencies will still be honored.</para>
         </listitem>
 
+        <para>If <literal>triggering</literal> is specified with
+        <command>systemctl stop</command>, the target unit is stopped
+        and any active units that trigger the target units are stopped.
+        This mode only works with <command>systemctl stop</command>. For
+        Triggers=, see
+        <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
```

Comment:
The first two sentences repeat the same thing. Also Triggers= should use `<varname>`.
And "target" has a special meaning in the context of units, so it should not be used here.
Maybe
```xml
<literal>triggering</literal> may only be used with <command>systemctl stop</command>.
In this mode, the specified unit and any active units that trigger it are stopped.
See the discussion of <varname>Triggers=</varname> in
<citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>
for more information about triggering units.
```